hack/test-integration.sh

```diff
@@ -28,6 +28,6 @@ echo "The following invocation will run all integration tests: "
 echo '    make test-integration'
 echo
 echo "The following invocation will run a specific test with the verbose flag set: "
-echo '    make test-integration WHAT=./test/integration/pods KUBE_GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"'
+echo '    make test-integration WHAT=./test/integration/pods GOFLAGS="-v" TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"'
```

Comment:
`KUBE_GOFLAGS` and `KUBE_TEST_ARGS` are used inside the scripts. We'd better let users use `GOFLAGS` and `TEST_ARGS` instead to pass params.